consensus/src/block/alt_block.rs

```diff
@@ -47,8 +47,10 @@ where
             BlockChainContextRequest,
             Response = BlockChainContextResponse,
             Error = tower::BoxError,
-        > + Send
-        + 'static,
+        >
+        + Send
+        + 'static
+        + ServiceExt<BlockChainContextRequest>,
```

Comment:
is this needed?

Reply:
At some point it just refused categorically from inferencing the `ServiceExt` trait so I had to add that but now that I've resolved all error It can be removed safely.